app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,16 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(self, hummer_level: int, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+        self._hummer_level = hummer_level
```

Comment:
The variable name 'hummer_level' seems to be a typo. If you meant 'hammer_level', please correct it to ensure clarity and correctness in variable naming.